### PR TITLE
Added missing ' (single quote) in error message in messages.py

### PR DESCRIPTION
**Pull request by BIGduzy**

### CODE REVIEW

**Review by tropxy — APPROVED** [review]

**Review by OrangeTux — DISMISSED** [review]
Thanks!

**Comment by BIGduzy** [comment]
Since opening the PR some changes where made and there now is a merge conflict. Should this be fixed on my end or will this be fixed by the code owners before the merge?

**Review by OrangeTux — APPROVED** [review]
Thanks!

**Pull request merged**
